releases/_posts/2019-04-23-spark-release-2-4-2.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Spark Release 2.4.2
+categories: []
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+
+Spark 2.4.2 is a maintenance release containing stability fixes. This release is based on the branch-2.4 maintenance branch of Spark. We strongly recommend all 2.4.0 users to upgrade to this stable release.
```

Comment:
very small nit - should `all 2.4.0` be `all 2.4.0 and 2.4.1` ?

Reply:
how about `all 2.4 users`?